pkg/apis/serving/v1beta1/inference_service_defaults.go

```diff
@@ -84,10 +89,26 @@ func (isvc *InferenceService) Default() {
 	if err != nil {
 		panic(err)
 	}
-	isvc.DefaultInferenceService(configMap, deployConfig)
+
+	if err := v1alpha1.AddToScheme(scheme.Scheme); err != nil {
+		mutatorLogger.Error(err, "unable to add v1alpha1 scheme")
+		panic(err)
+	}
+	var c client.Client
+	if c, err = client.New(cfg, client.Options{Scheme: scheme.Scheme}); err != nil {
+		mutatorLogger.Error(err, "Failed to start client")
+		panic(err)
+	}
+	models := &v1alpha1.ClusterLocalModelList{}
+	if err := c.List(context.TODO(), models); err != nil {
+		mutatorLogger.Error(err, "Cannot List local models")
+		panic(err)
```

Comment:
Todo: should not panic here.

Reply:
Well it should be fine, this is when it cannot check if the CRD is available. Not when the CR is not present.